Jenkinsfile.garbage_collection

```diff
@@ -0,0 +1,85 @@
+#!groovy
+
+import org.jenkinsci.plugins.workflow.steps.FlowInterruptedException
+// http://stackoverflow.com/questions/37425064/how-to-use-environment-variables-in-a-groovy-function-using-a-jenkinsfile
+import groovy.transform.Field
+@Field final BUILD_OS_TARGETS=['el7']
+
+@Field buildParams = [
+  "BUILD_OS": "all",
+  "REBUILD": "false",
+  "LEAVE_CONTAINER": "0",
+]
+def ask_build_parameter = { ->
+  return input(message: "Build Parameters", id: "build_params",
+    parameters:[
+      [$class: 'ChoiceParameterDefinition',
+        choices: "all\n" + BUILD_OS_TARGETS.join("\n"), description: 'Target OS name', name: 'BUILD_OS'],
+      [$class: 'ChoiceParameterDefinition',
+        choices: "false\ntrue", description: 'Rebuild cache image', name: 'REBUILD'],
+      [$class: 'ChoiceParameterDefinition',
+        choices: "0\n1", description: 'Leave container after build for debugging.', name: 'LEAVE_CONTAINER'],
+    ])
+}
+
+// Environment variables supplied by Jenkins system configuration:
+// env.REPO_BASE_DIR
+// env.BUILD_CACHE_DIR
+def write_build_env(label) {
+  def build_env="""# These parameters are read from bash and docker --env-file.
+# So do not use single or double quote for the value part.
+LEAVE_CONTAINER=${buildParams.LEAVE_CONTAINER}
+REPO_BASE_DIR=${env.REPO_BASE_DIR ?: ''}
+BUILD_CACHE_DIR=${env.BUILD_CACHE_DIR ?: ''}
+BUILD_OS=${label}
+REBUILD=${buildParams.REBUILD}
+RELEASE_SUFFIX=${RELEASE_SUFFIX}
+# https://issues.jenkins-ci.org/browse/JENKINS-30252
+GIT_BRANCH=${env.BRANCH_NAME}
+BRANCH_NAME=${env.BRANCH_NAME}
+BRANCH=${env.BRANCH_NAME}
+"""
+  writeFile(file: "build.env", text: build_env)
+}
```

Comment:
~~I haven't read the actual cleanup script yet but this looks like code copy-pasted from the main Jenkinsfile. I think there are many variables here that are not used. For example there is `LEAVE_CONTAINER` but I don't think this job starts any containers.~~

~~To keep the code clean and readable, any unused variables should be removed.~~

I see below that a copy of this file was committed with the code cleaned up. I guess you just forgot to call `git rm` on this.